docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_BERT_From_Tensorflow.md

```diff
@@ -35,7 +35,6 @@ To generate the BERT Intermediate Representation (IR) of the model, run the Mode
 python3 ./mo_tf.py
 --input_meta_graph uncased_L-12_H-768_A-12/bert_model.ckpt.meta \
 --output bert/pooler/dense/Tanh                                 \
---disable_nhwc_to_nchw                                          \
```

Comment:
are we sure that all of these supported models are working now?

Reply:
Yes, we tested these models